client/jansson/load.c

```diff
@@ -114,7 +114,7 @@ static void error_set(json_error_t *error, const lex_t *lex,
         if(saved_text && saved_text[0])
         {
             if(lex->saved_text.length <= 20) {
-                snprintf(msg_with_context, JSON_ERROR_TEXT_LENGTH,
+                snprintf(msg_with_context, JSON_ERROR_TEXT_LENGTH + 17,
```

Comment:
why +17? saved_text can be up to 20, and we are adding " near ''" (8 characters).

Reply:
Yes 8 characters are enough.

Reply:
No, you got me wrong. 8 for the " near to ''" + 20 for saved_text = 28.